build.sbt

```diff
@@ -29,8 +29,8 @@ lazy val root = (project in file("."))
         exclude ("com.fasterxml.jackson.module", "jackson-module-scala_2.13")
         exclude ("com.typesafe", "ssl-config-core_2.13"),
       "org.postgresql" % "postgresql" % "42.7.3",
-      "com.okta.sdk" % "okta-sdk-api" % "16.0.0",
-      "com.okta.sdk" % "okta-sdk-impl" % "16.0.0" % Runtime,
+      "com.okta.sdk" % "okta-sdk-api" % "15.0.0",
+      "com.okta.sdk" % "okta-sdk-impl" % "15.0.0" % Runtime,
```

Comment:
we are going down ?

Reply:
I was testing this and realised that the latest main build hasn't worked since #46 so it's either go down or bring identity-auth-core up.